.gitignore

```diff
@@ -127,4 +127,7 @@ dist
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
+
+# Mac artifacts
+.DS_Store
```

Comment:
Doesn't matter so much here, and we _are_ a mac house...
but I reckon things like `.DS_Store` better sit in a global excludes file specific to operating https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Can have a chat about first time it comes up on a prod app?